src/cider/nrepl/middleware/util/misc.clj

```diff
@@ -17,8 +17,15 @@
   (not (nil? (boot-fake-classpath))))
 
 (def java-api-version
-  (try (-> (System/getProperty "java.version") (str/split #"\.") second)
-       (catch Exception _ "7")))
+  (try
+    (let [java-ver (System/getProperty "java.version")
+          splited-ver (str/split java-ver #"\.")
```

Comment:
Why don't you just using destructuring here? I'd also name `old-ver` and `new-ver` `major` and `minor`, as that's what they truly are.

Reply:
you are right. I did not think :), I will modify this. 
thanks for your checking.

Reply:
you are right, but versioning is changed from java9, so I use new and old :)
anyway, I will fix it as your comment. I will send new pull request. 
thanks for your checking.